### DISCUSSION TITLE
Sizing experience

### DISCUSSION THREAD

**Discussion by HachimanSec** [telekom-security/tpotce#1082] [General]
Hi everyone, 

just a FYI for the community. 
I deployed my T-Pot instance on a Digital Ocean droplet with **4 vCPUs, 8GB RAM, 160GB Disk**. 

It keeps crashing / becomes unresponsive (SSH, Web) once a day and I have to power cycle the machine. 

I deployed the standard version of T-Pot and didn't made any customisations, as it is just a test / play system.

**Comment by t3chn0m4g3** [comment]
@HachimanSec The recommendation for `Standard` is [8-16GB](https://github.com/telekom-security/tpotce#system-requirements) of RAM.

With regard to the hoster, as described [here](https://github.com/telekom-security/tpotce#running-in-a-cloud), to some degree you might need modifications, especially should RAM be the issue.

To pinpoint the reason I recommend to check out the [Troubleshooting Section](https://github.com/telekom-security/tpotce#troubleshooting).

**Reply by HachimanSec** [reply]
Thanks for the feedback @t3chn0m4g3 . Wasnt really meant to be a complaint. Just an info when someone tries to target the low end of the specs. I will def. ramp up the RAM and see then.

**Reply by t3chn0m4g3** [reply]
I did not see it as a complaint, but wanted to make sure if others read the the thread they have orientation for troubleshooting and such.
